Add a contract customization example to the Required members documentation (#31555)
dotnet/docs: docs/standard/serialization/system-text-json/required-properties.md

    public int Age { get; set; }
}
```

> It is also possible to control whether a property is required via the contract model using the <xref:System.Text.Json.Serialization.Metadata.JsonPropertyInfo.IsRequired?displayProperty=nameWithType> property:

Reviewer: ```suggestion
It's also possible to control whether a property is required via the contract model using the <xref:System.Text.Json.Serialization.Metadata.JsonPropertyInfo.IsRequired?displayProperty=nameWithType> property:
```

```csharp
var options = new JsonSerializerOptions
{
    TypeInfoResolver = new DefaultJsonTypeInfoResolver
    {
        Modifiers =
        {
            static typeInfo =>
            {
                if (typeInfo.Kind != JsonTypeInfoKind.Object)
                    return;

                foreach (JsonPropertyInfo propertyInfo in typeInfo.Properties)
                {

>                     // strip IsRequired constraint from every property

Reviewer 2: ```suggestion
                    // Strip IsRequired constraint from every property.
```

                    propertyInfo.IsRequired = false;
                }
            }
        }
    }
};

> JsonSerializer.Deserialize<Person>("""{"Age": 42}""", options); // serialization now succeeds

Reviewer 2: ```suggestion
// Deserialization now succeeds even though the Name property isn't in the JSON payload.
JsonSerializer.Deserialize<Person>("""{"Age": 42}""", options);
```
